app/models/document.rb

```diff
@@ -27,8 +27,15 @@ class Document < ApplicationRecord
     reviewed
   ].freeze
 
+  LIVE_STATES = %w[
+    published
+    retired
+    removed
+  ].freeze
+
   validates_inclusion_of :publication_state, in: PUBLICATION_STATES
   validates_inclusion_of :review_state, in: REVIEW_STATES
+  validates_inclusion_of :live_state, in: LIVE_STATES, allow_nil: true
```

Comment:
Might want to only allow_nil when it's not sent_to_live

Reply:
Updated and fixed-up